### PR TITLE
front: fix scenario infralist error

### LINKED ISSUE #4487: front: when the request to get the infra list fails, no error is reported
### What happened?

When creating a scenario, the request for the infra list failed (400, can't connect to core/cache_status/).

The request list was displayed as empty, with no indication that something went wrong, implying that the database had no infra.

### What did you expect to happen?

A clear error message

### How can we reproduce it (as minimally and precisely as possible)?

1. Load all services except core (which would cause this error or a similar one)
2. Try to create a scenario

### What operating system, browser and environment are you using?

- **Browser**: Firefox 114.0.2
- **OS**: Windows 10
- **Env**: Local

### OSRD version (top right corner `Account` button > `Informations`)

6807bd9625269dae95a2608625fe2bd4be8535d9

### PR DESCRIPTION
**Pull request by SharglutDev**
fix issue in scenario where no error was displayed if infralist coudln't be fetched

close #4487

### CODE REVIEW

File: front/src/modules/scenario/components/AddOrEditScenarioModal.tsx [hunk 1]
```diff
@@ -47,14 +47,17 @@ export default function AddOrEditScenarioModal({
   };
 
   const [deleteScenarioRTK] =
```
Comment (Akctarus) [inline front/src/modules/scenario/components/AddOrEditScenarioModal.tsx]: you can remove the 'RTK' here

**Review by Akctarus — APPROVED** [review]
Tested and it works well ! Ty for this fix

File: front/src/modules/scenario/components/AddOrEditScenarioModal.tsx (hunk 1 above)
Comment (SharglutDev) [reply front/src/modules/scenario/components/AddOrEditScenarioModal.tsx]: I can but had to change the deleteScenario event function name

**Review by clarani — APPROVED** [review]
Thanks for renaming all the endpoints :pray: Nice PR ! tested too :heavy_check_mark:

**Pull request merged**